src/main/java/nl/praegus/fitnesse/symbols/MavenProjectVersions/VersionStatus.java

```diff
@@ -0,0 +1,7 @@
+package nl.praegus.fitnesse.symbols.MavenProjectVersions;
+
+public enum VersionStatus {
+    Outdated,
+    Up_to_date,
+    Ahead
+}
```

Comment:
Enum values schrijf je in principe in java altijd alleen met hoofdletters en underscors. Je kan er ook mooi de string value in kwijt, op deze manier: 
```
public enum VersionStatus {
    OUTDATED("Outdated"),
    UP_TO_DATE("Up-to-date"),
    AHEAD("Ahead");

    private final String stringValue;

    VersionStatus(String envUrl) {
        this.stringValue = envUrl;
    }

    @Override
    public String toString() {
        return stringValue;
    }
}
```

Dan hoef je ook niet op de plek waar je deze gebruikt die conversie te doen.

Reply:
Enum klasse geoptimaliseerd zoals aangegeven